Index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+
+    <nav id="navbar">
+        <header>Technical documentation</header>
+        <ul>
+            <li><a class="nav-link" href="Introduction">Introduction</a></li>
+            <li><a class="nav-link" href="What_you_should_know">What you should know</a></li>
+            <li><a class="nav-link" href="While_Statement">While Statement</a></li>
+            <li><a class="nav-link" href="Hello_World">Hello World</a></li>
+            <li><a class="nav-link" href="Variables">Variables</a></li>
+
+        </ul>
+    </nav>
+
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<main id="main-doc">Technical Documentation
+    <br>
+    <section class="main-section" id="Introduction">
```

Comment:
When using the <section> tag kindly make sure that it has a closing tag after the section, in order to close the section.

Reply:
- [ ] Be specific , kindly use emojis also for human friendly reason :+1:  :smile: